client/src/components/Navbar/Navbar.tsx

```diff
@@ -114,7 +114,7 @@ const Navbar: React.FC = () => {
   const renderMenuItems = () => {
     // TODO: conditionally render based on profile type
     return menuItems.map((menu) => {
-      if (menu.authenticated) {
+      if (menu.authenticated && menu.canView) {
```

Comment:
This isn't at all the expectation. We need to check on the type of account signed in see if their authorized for that page. Pausing this as we are going to be separating the pet sitter and customers.